pom.xml

```diff
@@ -895,6 +780,131 @@
 			</plugin>
 		</plugins>
 	</build>
+
+	<!-- Set reports for the maven site plugin -->
+	<reporting>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-project-info-reports-plugin</artifactId>
+				<version>2.9</version>
+				<reportSets>
+					<reportSet>
+						<reports>
+							<report>index</report>
+							<report>dependencies</report>
+							<report>dependency-convergence</report>
+							<report>dependency-management</report>
+							<report>plugins</report>
+						</reports>
+					</reportSet>
+		        </reportSets>
+			</plugin>
+
+			<!-- PMS tests report -->
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-surefire-report-plugin</artifactId>
+				<version>2.19.1</version>
+				<configuration>
+					<showSuccess>false</showSuccess>
+				</configuration>
+			</plugin>
+
+			<!-- Javadoc -->
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-javadoc-plugin</artifactId>
+				<version>${maven-javadoc-plugin-version}</version>
+				<configuration>
+					<quiet>true</quiet>
+					<tags>
+						<tag>
+							<name>todo</name>
+							<placement>a</placement>
+							<head>To Do:</head>
+						</tag>
+						<tag>
+							<name>review</name>
+							<placement>a</placement>
+							<head>Review:</head>
+						</tag>
+					</tags>
+					<detectJavaApiLink>false</detectJavaApiLink>
+					<links>
+						<link>http://docs.oracle.com/javase/7/docs/api/</link>
+					</links>
+
+					<!--
+						Doclet to generate class diagrams inside Javadoc.
+						Requires the installation of Graphviz (http://www.graphviz.org/)
+					-->
+					<doclet>org.jboss.apiviz.APIviz</doclet>
+					<docletArtifact>
+						<groupId>org.jboss.apiviz</groupId>
+						<artifactId>apiviz</artifactId>
+						<version>1.3.2.GA</version>
+					</docletArtifact>
+					<useStandardDocletOptions>true</useStandardDocletOptions>
+				</configuration>
+			</plugin>
+
+			<!-- Class dependency report -->
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>jdepend-maven-plugin</artifactId>
+				<version>2.0</version>
+			</plugin>
+
+			<!-- Findbugs report -->
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>findbugs-maven-plugin</artifactId>
+				<version>3.0.4</version>														
+			</plugin>
+
+			<!-- Checkstyle report -->
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-checkstyle-plugin</artifactId>
+				<version>2.17</version>
```

Comment:
For that specific case, is that `${mavenCheckstylePluginVersion}` instead of a nominative version, will make the POM more easily maintenable without having any secondary negative effects ?

Though perhaps it could be extended to all report plugins without negative consequences ?